### PR TITLE
yelp place details method

### PR DESCRIPTION
**Pull request by jordanskole**
@kennethkoontz please review

### CODE REVIEW

**Comment by kennethkoontz** [comment]
:+1:  nice!

File: benchmarkplaces/providers.py [hunk 1]
```diff
@@ -163,6 +163,23 @@ def search_places(self, name, address, **kwargs):
         else:
             raise APIError('An error occurred with %s API' % self.name)
 
+    # yelp place detail request
+    def get_yelp_details(self, yelp_id, **kwargs):
+        url = 'http://api.yelp.com/v2/business/' + yelp_id
```
Comment (kennethkoontz) [inline benchmarkplaces/providers.py:168]: I normally do string interpolation here.

`url = 'http://api.yelp.com/v2/business/%s' % yelp_id`

Comment (kennethkoontz) [inline benchmarkplaces/providers.py:168]: But not a biggy.

Comment (jordanskole) [inline benchmarkplaces/providers.py:168]: Ohhh okay, that makes more sense because of python weirdness i've noticed, so i'll do it that way from now on.

**Pull request merged**